test/app.js

```diff
@@ -0,0 +1,117 @@
+var harvester = require('../lib/harvester');
+var JSONAPI_Error = harvester.JSONAPI_Error;
+var Promise = require('bluebird');
+var config = require('./config.js');
+
+function configureDefaultApp(app) {
```

Comment:
This fn is needed if the plan is that you keep all applications in one file, and you plan to have [DefaultTestApp, OplogTestApp, etc] and you want to make generating them easy from that one file. However I think it makes more sense to simple have 3 separate application files, where one is DefaultTestApp, the next is OplogTestApp, etc, and you're able to use those applications as needed in specific tests, with the defaultTestApp always running and the others being initiable by tests.